test_resource/test_repo/src/Python_annot_file.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function
```

Comment:
The file naming start with a caps : Python_, Which is not standard across other file naming